README.md

```diff
@@ -252,7 +252,7 @@ The following files are also included unless the `--skip-tests` is used:
 		default: travis
 		options:
 		  - travis
-		  - circle
+		  - circleci
```

Comment:
This shouldn't be changed I think for BC.

Reply:
BC?

Reply:
Sorry, backward compatibility.

Reply:
Ah. Makes sense. I'll revert shortly.